src/ansys/platform/instancemanagement/__init__.py

```diff
@@ -50,7 +50,7 @@ def is_configured() -> bool:
     Returns
     -------
     bool
-       ``True`` when the environment is configured to use PyPIM, ``False`` when it is not.
+        ``True`` when the environment is configured to use PyPIM, ``False`` otherwise.
```

Comment:
I'm hoping that this missing space is what is causing the odd rendering in the documentation.